spring-webmvc/src/main/java/org/springframework/web/servlet/config/annotation/DelegatingWebMvcConfiguration.java

```diff
@@ -46,10 +47,8 @@
 
 
 	@Autowired(required = false)
-	public void setConfigurers(List<WebMvcConfigurer> configurers) {
-		if (!CollectionUtils.isEmpty(configurers)) {
-			this.configurers.addWebMvcConfigurers(configurers);
-		}
+	public void setConfigurers(ObjectProvider<WebMvcConfigurer> configurers) {
```

Comment:
Autowiring a list already orders it the same way `ObjectProvider#orderedStream` does. Do you have any indication that's not the case? If so, adding a test that showcases that is necessary.